src/events/http/HttpServer.js

```diff
@@ -906,9 +906,11 @@ export default class HttpServer {
         const apiKey = createApiKey()
 
         this.#apiKeysValues = getApiKeysValues(
-          this.#serverless.service.provider.apiGateway?.apiKeys ?? [apiKey],
+          this.#serverless.service.provider.apiGateway?.apiKeys ?? [],
         )
 
+        this.#apiKeysValues.push(apiKey)
```

Comment:
getApiKeysValues is returning a Set, where `push` does not exist. we would need to use `.add` here.